commands/quiz.js

```diff
@@ -16,6 +32,7 @@ module.exports = {
         const question = randomQuestion.question;
         const options = randomQuestion.options;
         const answer = randomQuestion.answer;
+        const explaination = `Because it is what it is`;
```

Comment:
`explanation` (spelled wrong), also should we have this as part of the questions.json? Not sure if you already did this, but makes sense to me. It can be optional.

Reply:
Sorry boss. Won't happen again, I promise.